### PR TITLE
update NuGetGallery submodule hash

### PR DESCRIPTION
**Pull request by drewgillies**
Updating the submodule hash for completeness after removing the myget feed from NuGetGallery. Not an entirely necessary change given that the submodule isn't built--I believe a UI dependencies are all that are used. To test I've just ensured that basic pages have come up. If we feel that this introduces unnecessary risk, let's abandon it. If it breaks anything, we can revert it.

### CODE REVIEW

File: src/NuGet.Status/Web.config [hunk 1]
```diff
@@ -22,16 +22,15 @@
...
     <add key="NuGetBaseUrl" value="https://dev.nugettest.org"/>
-    <add key="Ida:AdminEnabled" value="false"/>
-    <add key="Ida:RootUri" value=""/>
-    <add key="Ida:AADInstance" value=""/>
-    <add key="Ida:ClientId" value=""/>
-    <add key="Ida:Tenant" value=""/>
-    <add key="ExternalBrandingMessage" value="© Microsoft 2019"/>
+    <add key="Ida:AdminEnabled" value="true"/>
+    <add key="Ida:RootUri" value="https://localhost"/>
+    <add key="Ida:AADInstance" value="https://login.microsoftonline.com/"/>
```
Comment (ryuyu) [inline src/NuGet.Status/Web.config:27]: should probably remove these values.

**Comment by joelverhagen** [comment]
The submodule change is not showing in the diff. I don't know if that's expected or not.

File: src/NuGet.Status/Web.config [hunk 2]
```diff
@@ -22,16 +22,15 @@
...
+    <add key="Ida:ClientId" value="75891a33-35a9-4a88-9552-5add848119f8"/>
+    <add key="Ida:Tenant" value="72f988bf-86f1-41af-91ab-2d7cd011db47"/>    <add key="ExternalBrandingMessage" value="© Microsoft 2019"/>
```
Comment (joelverhagen) [inline src/NuGet.Status/Web.config:29]: New line between the two `<add>` elements.

File: src/NuGet.Status/Web.config [hunk 3]
```diff
@@ -22,16 +22,15 @@
...
     <add key="ExternalPrivacyPolicyUrl" value="https://go.microsoft.com/fwlink/?LinkId=521839"/>
     <add key="TrademarksUrl" value="https://www.microsoft.com/trademarks"/>
     <add key="ApplicationInsightsKey" value=""/>
-    <add key="AdminIdentities" value=""/>
+    <add key="AdminIdentities" value="drewgil@microsoft.com"/>
```
Comment (joelverhagen) [inline src/NuGet.Status/Web.config:33]: This should be cleared out.

**Review by joelverhagen — APPROVED** [review]
I don't see a submodule change, but maybe it's a GitHub PR viewing issue.

**Comment by drewgillies** [comment]
@ryuyu @joelverhagen Let's start again--now we have the submodules update and the test values removed.

**Comment by joelverhagen** [comment]
I see it now! :shipit:

**Review by ryuyu — APPROVED** [review]

**Pull request merged**